fargate/cloudformation/batch-fargate-new-vpc.yaml

```diff
@@ -0,0 +1,357 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I believe core infrastructure is generic and should belong into a `core_infra/ec2-fargate/cloudformation` folder.

Reply:
No problem. Should I separate the “hello world” job definitions to this project and call out to the core infra as an additional stack with exports?

Reply:
Please call out the core infra as a step in the process.
You can leave job definition in IaC.